shopelectro/tests/tests_html.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+from django.test import TestCase, tag
+from django.conf import settings
+from django.template.loader import render_to_string
+
+from html5validate import validate
+from html5lib import html5parser
+
+
+@tag('fast')
+class TemplateTests(TestCase):
+
+    @unittest.skip('should fix html templates')
+    def test_templates(self):
+        for dir, _, filenames in os.walk(settings.TEMPLATE_DIR):
```

Comment:
`settings.TEMPLATE_DIR` is redundant option, because we'll not redefine it for different envs and django extensions. You can move it to the module level const or just hardcode there.
We try to keep django settings files as small as possible to reduce problems with tuning prod/local envs

Reply:
did you mean TEMPLATE_ASSETS_DIR?

Reply:
@olegush , y, i meant it, sorry